pyanaconda/modules/payloads/payload/flatpak/flatpak_manager.py

```diff
@@ -233,21 +239,35 @@ def install(self, progress: ProgressReporter):
         log.debug("Installing Flatpaks")
 
         installation = self._create_flatpak_installation()
-        self._transaction = self._create_flatpak_transaction(installation)
+        saved_urls = None
 
-        if self._collection_location:
-            self._transaction.add_sideload_image_collection(self._collection_location, None)
+        try:
+            if self._collection_location:
+                # If we're installing from a local source, we need to disable
+                # loading the list of current Flatpaks from the network - we
+                # want to install what was in the install image even if it is
+                # out of date.
```

Comment:
Maybe we should include a justification why we want that ? Its a bit counter intuitive, as for both Fedora mirrors and Red Hat CDN or Satellite we do default to fetching the latest (and presumably more secure) RPMs from the network even if a local DVD source is available.